apps/frontend/src/app/5_pages/BitocracyPage/components/ProposalTitle/ProposalTitle.tsx

```diff
@@ -1,19 +1,30 @@
-import React, { FC } from 'react';
+import React, { FC, useMemo } from 'react';
 
 import { Tooltip, TooltipTrigger } from '@sovryn/ui';
 
 import { ProposalProps } from '../../BitocracyPage.types';
 
-export const ProposalTitle: FC<ProposalProps> = ({ proposal }) => (
-  <Tooltip
-    trigger={TooltipTrigger.click}
-    className="sm:text-base cursor-pointer"
-    tooltipClassName="break-words"
-    content={<>{proposal.description}</>}
-    children={
-      <div>
-        <p className="truncate max-w-[20rem] m-0">{proposal.description}</p>
-      </div>
+export const ProposalTitle: FC<ProposalProps> = ({ proposal }) => {
+  const title = useMemo(() => {
+    let title = proposal?.description.slice(0, 50);
+    if (proposal && proposal?.description.length > 50) {
+      title += '...';
     }
-  />
-);
+
+    return title;
+  }, [proposal]);
```

Comment:
```
  const title = useMemo(() => {
    if (proposal?.description.length <= titleCharacterLimit) {
      return proposal?.description;
    }

    return `${proposal?.description.slice(0, titleCharacterLimit)}...`;
  }, [proposal]);
```
Define `titleCharacterLimit` (`50`) above the component, it's better than using magic numbers.